MdeModulePkg/Universal/CapsulePei/CapsuleX64.inf

```diff
@@ -50,3 +50,11 @@
 
 [UserExtensions.TianoCore."ExtraFiles"]
   CapsuleX64Extra.uni
+
+## MS_CHANGE_211478

```

Comment:
Can you please change these to `MU_CHANGE`?

Reply:
Updated.